qemu/tests/cfg/fio_perf.cfg

```diff
@@ -0,0 +1,78 @@
+- fio:
+    only x86_64
+    type = fio_perf
+    numa_node = -1
+    block_size = "4k 16k 64k 256k"
+    format = False
+    iodepth = "1 8 64"
+    threads = "16"
+    rw = "read write randread randwrite randrw"
+    variants:
+        - localfs:
+            image_name_disk1 = /mnt/storage2
+        - nfs_block:
+            image_name_disk1 = images/storage2
+        - fusion-io:
+            image_name_disk1 = /dev/fioa
+    variants:
+        - file_system:
+            image_size_disk1 = 40G
+            force_create_image_disk1 = yes
+            remove_image_disk1 = yes
+            format = True
+        - raw_disk:
+            image_raw_device_disk1 = yes
+            vhost_nic1 =
+    variants:
+        - single_disk:
+            images += " disk1"
+
+    variants:
```

Comment:
No need for this variants,
- fio:
    only x86_64
...
    rw = "read write randread randwrite randrw"
    Linux:
        pattern = ...
        order_list = ...
    Windows:
        pattern = ...
        order_list = ...